recipes/fc/meta.yaml

```diff
@@ -39,7 +39,7 @@ about:
   
 test:
   commands:
-    - $./bin/fc-virus --help
+    - ${PREFIX}/bin/fc-virus --help
```

Comment:
Could you update to`- fc-virus --help` rather than `- ${PREFIX}/bin/fc-virus --help`